actors_tools/GlobalActorReader/GlobalActorReader.cc

```diff
@@ -156,6 +161,16 @@ TVector3 GlobalActorReader::GetPolarizationAfterProcess() {return PolarizationAf
 
 std::string GlobalActorReader::GetProcessName() {return ProcessName.GetValue();}
 
+int GlobalActorReader::GetParentID() {return ParentID.GetValue();}
```

Comment:
shouldnt be those methods const?
GetParentID() const {return ...}

Reply:
Done